pkg/apis/mxnet/v1/groupversion_info.go

```diff
@@ -0,0 +1,34 @@
+// Copyright YEAR The Kubeflow Authors
```

Comment:
Will we modify script to change `YEAR` to `2021` in the future ?

Reply:
Yeah, I document this issue here https://github.com/kubeflow/tf-operator/issues/1309.

Reply:
Thanks!